lib/mediawiki.ParsoidConfig.js

```diff
@@ -136,6 +136,8 @@ function ParsoidConfig(localSettings, options) {
 		performanceTimer: true,
 		loggerBackend: true,
 		nativeExtensions: true,
+		mwApiMap: true,
+		reversemwApiMap: true,
```

Comment:
The new version of Parsoid freezes the configuration so that it cannot be changed dynamically. This tells the config to ignore these fields when freezing the object.